src/fiatExchanges/FiatExchangeAmount.test.tsx

```diff
@@ -149,6 +153,23 @@ describe('FiatExchangeAmount cashIn', () => {
     const mockScreenProps = getMockStackScreenProps(Screens.FiatExchangeAmount, {
       currency: CiCoCurrency.cEUR,
       flow: CICOFlow.CashIn,
+      network: Network.Celo,
+    })
+    const tree = render(
+      <Provider store={storeWithUSD}>
+        <FiatExchangeAmount {...mockScreenProps} />
+      </Provider>
+    )
+
+    fireEvent.changeText(tree.getByTestId('FiatExchangeInput'), '0')
+    expect(tree.getByTestId('FiatExchangeNextButton')).toBeDisabled()
+  })
+
+  it('disables the next button if the ETH amount is 0', () => {
```

Comment:
seems like we should do this with an it.each since it largely copies some similar tests